cartographer_ros/cartographer_ros/node_options.cc

```diff
@@ -34,6 +37,14 @@ NodeOptions CreateNodeOptions(
       lua_parameter_dictionary->GetDouble("submap_publish_period_sec");
   options.pose_publish_period_sec =
       lua_parameter_dictionary->GetDouble("pose_publish_period_sec");
+  options.trajectory_publish_period_sec =
+      lua_parameter_dictionary->HasKey("trajectory_publish_period_sec")
+          ? lua_parameter_dictionary->GetDouble("trajectory_publish_period_sec")
+          : kDefaultTrajectoryPublishPeriodSec;
+  options.trajectory_hue_first =
```

Comment:
I thought it would be nice if you could specify the color of the trajectory. You may want to color the trajectories of different robots (specified with different config files) differently.